### ISSUE TITLE
[FEATURE] Change pulsar-flink-connector-${scala.binary.version}-${flink.binary.version} to flink-connector-pulsar_${scala.binary.version}

### ISSUE THREAD

**Issue by sharkdtu** [streamnative/pulsar-flink#216]
Can we change the artifact-id to flink-connector-pulsar_${scala.binary.version} whose branch is flink-${flink.binary.version}?  I think it is more in line with the style of the flink community, and we can contribute this connector to flink main project in the feature.

**Comment by reswqa** [comment]
Yes, you have a point, but we are currently using a set of styles maintained internally by StreamNative. We are currently talking to the Flink community about merge, and if we maintain our project in Flink after they agree, the issues will be the same as you want them to be.

**Comment by jianyun8023** [comment]
@sharkdtu We have now adopted a similar rule, the new artifactId `pulsar-flink-connector_${scala.binary.version}`, version is `${flink_version}.${build_number}`.
We think this structure is a more friendly solution.

**Comment by jianyun8023** [comment]
We need to maintain multiple flink versions of the connector, but each minor_version has incompatible differences.
